ruff.toml

```diff
@@ -0,0 +1,25 @@
+# Support Python 3.10+.
+target-version = "py310"
+# Set the maximum line length to 79.
```

Comment:
This seems wrong

Reply:
It's based on the configuration documentation from ruff https://docs.astral.sh/ruff/settings/#target-version , and I removed it as having a set `requires-python` in `pyproject.toml` is enough